tests/schemas/invalid/test_invalid_date_range_period.json

```diff
@@ -2,74 +2,59 @@
   "data_version": "0.0.3",
   "description": "A test schema for different date formats",
   "legal_basis": "StatisticsOfTradeAct",
-  "metadata": [
-    {
-      "name": "user_id",
-      "type": "string"
-    },
-    {
-      "name": "ru_name",
-      "type": "string"
-    },
-    {
-      "name": "period_id",
-      "type": "string"
-    }
-  ],
+  "metadata": [{
+    "name": "user_id",
+    "type": "string"
+  }, {
+    "name": "ru_name",
+    "type": "string"
+  }, {
+    "name": "period_id",
+    "type": "string"
+  }],
   "mime_type": "application/json/ons/eq",
   "schema_version": "0.0.1",
-  "sections": [
-    {
-      "groups": [
-        {
-          "blocks": [
-            {
-              "id": "date-range-block",
-              "question": {
-                "answers": [
-                  {
-                    "id": "date-range-from",
-                    "label": "Period from",
-                    "mandatory": true,
-                    "type": "Date"
-                  },
-                  {
-                    "id": "date-range-to",
-                    "label": "Period to",
-                    "mandatory": true,
-                    "type": "Date"
-                  }
-                ],
-                "id": "date-range-question",
-                "period_limits": {
-                  "maximum": {
-                    "days": 20,
-                    "months": 1
-                  },
-                  "minimum": {
-                    "days": 23,
-                    "years": 1
-                  }
-                },
-                "title": "Enter Date Range",
-                "type": "DateRange"
-              },
-              "title": "Date Range",
-              "type": "Question"
+  "sections": [{
```

Comment:
How has this changed? Is yarn format not able to correct it?

Reply:
I see. It was broken

Reply:
yeh this is the output from the format command